scc/src/opts.rs

```diff
@@ -0,0 +1,147 @@
+use bpaf::*;
+use std::ffi::OsString;
+use std::path::PathBuf;
+use std::collections::VecDeque;
+
+#[cfg(test)]
+use std::{println as info};
+
+pub fn fix_args(args: Vec<String>) -> Vec<String> {
+    let mut fixed_args: Vec<String> = vec![];
+    let mut broken = false;
+
+    #[cfg(test)]
+    info!("Raw args: {:#?}", args);
+
+    // when cyberpunk's args are processed, the \" messes up the grouping, so we need to fix the args that have quotes & spaces in them
+    for arg in args {
+        let contains_quote = arg.contains('"');
+        if !contains_quote && !broken {
+            fixed_args.push(arg.clone());
```

Comment:
this doesn't need to clone
```suggestion
            fixed_args.push(arg);
```